### ISSUE TITLE
Ensure that model training can be restarted from a checkpoint if it breaks

### ISSUE THREAD

**Issue by KennethEnevoldsen** [centre-for-humanities-computing/danish-foundation-models#125]
Currently, learning rate decay along with place in the training data can not be recovered all need to be manually approximated.

**Comment by KennethEnevoldsen** [comment]
This was added in #155
